lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -422,6 +422,11 @@ class MailboxDashBoardController extends ReloadableController {
         && dashboardRoute.value == DashboardRoutes.emailDetailed;
   }
 
+  void clearSearchEmail() {
+    dispatchAction(ClearSearchEmailAction());
```

Comment:
by the way, what is different between action and state ?

Reply:
in common: 
- `*Action` is the state, but it come from the user interactions (tap on button, click, drag, ....sth like that)